### ISSUE TITLE
Filter by Price value is truncated when Price value is more than 4 digits.

### ISSUE THREAD

**Issue by gglobalstep** [woocommerce/woocommerce-gutenberg-products-block#2752]
## Description:
Filter by Price value is truncated when Price value is more than 4 digits.

## Environment:
**Windows 10**
- Browser: Chrome(Version 83.0.4103.61 (64-bit))
- Browser: Firefox(Version 77.0.1)(64-bit)

**MacBook Pro Version 10.15.5 (19F101)**
- Browser: Safari(Version 13.1.1)(15609.2.9.1.2)

## Steps to Reproduce:
1. Launch any site from the 5 test Sites.
1. Login with valid credentials.
1. Go to Pages->Add New.
1. Add All Products block.
1. Add Filter by Price block.
1. Click on Preview.
1. Set filter by price values more than 4 digits.
1. Observe that filter by Price displays truncated.

## Actual Result:
Filter by Price displays truncated when Price value is in thousands and above.

## Expected Result:
Filter by Price should display properly even if price value is more than 4 digits.

## Artifact:
![#26813](https://user-images.githubusercontent.com/41110392/85047227-25375d80-b1af-11ea-93b6-8f4224d32dd8.gif)

**Isolating the problem (mark completed items with an [x]):**
- [ ] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [ ] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**WordPress Environment**
<details>
```
`
### WordPress Environment ###

WordPress address (URL): https://bluehost-and-a8c.com
Site address (URL): https://bluehost-and-a8c.com
WC Version: 4.3.0
REST API Version: ✔ 1.0.10
WC Blocks Version: ✔ 2.7.1
Action Scheduler Version: ✔ 3.1.6
WC Admin Version: ✔ 1.3.0-beta.1
Log Directory Writable: ✔
WP Version: 5.4.2
WP Multisite: –
WP Memory Limit: 256 MB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: Apache
PHP Version: 7.2.31
PHP Post Max Size: 260 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.70.0
OpenSSL/1.1.1g

SUHOSIN Installed: –
MySQL Version: 5.7.23-23
Max Upload Size: 256 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 4.3.0
WC Database Prefix: _DYD_
Total Database Size: 6.92MB
Database Data Size: 5.98MB
Database Index Size: 0.94MB
_DYD_woocommerce_sessions: Data: 0.02MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_api_keys: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_attribute_taxonomies: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_downloadable_product_permissions: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_woocommerce_order_items: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_order_itemmeta: Data: 0.03MB + Index: 0.03MB + Engine MyISAM
_DYD_woocommerce_tax_rates: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_woocommerce_tax_rate_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_shipping_zones: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_shipping_zone_locations: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_shipping_zone_methods: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_payment_tokens: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_payment_tokenmeta: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_woocommerce_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_actionscheduler_actions: Data: 0.25MB + Index: 0.07MB + Engine MyISAM
_DYD_actionscheduler_claims: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_actionscheduler_groups: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_actionscheduler_logs: Data: 0.15MB + Index: 0.09MB + Engine MyISAM
_DYD_commentmeta: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_comments: Data: 0.16MB + Index: 0.13MB + Engine MyISAM
_DYD_email_log: Data: 0.03MB + Index: 0.00MB + Engine MyISAM
_DYD_links: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_mailchimp_carts: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_mailchimp_jobs: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_ms_snippets: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_options: Data: 4.65MB + Index: 0.15MB + Engine MyISAM
_DYD_postmeta: Data: 0.43MB + Index: 0.18MB + Engine MyISAM
_DYD_posts: Data: 0.22MB + Index: 0.13MB + Engine MyISAM
_DYD_snippets: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_termmeta: Data: 0.01MB + Index: 0.01MB + Engine MyISAM
_DYD_terms: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_term_relationships: Data: 0.01MB + Index: 0.03MB + Engine MyISAM
_DYD_term_taxonomy: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_usermeta: Data: 0.02MB + Index: 0.01MB + Engine MyISAM
_DYD_users: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_wc_admin_notes: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_wc_admin_note_actions: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_wc_category_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_wc_customer_lookup: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_wc_download_log: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_wc_order_coupon_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_wc_order_product_lookup: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_wc_order_stats: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_wc_order_tax_lookup: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_wc_product_meta_lookup: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_wc_reserved_stock: Data: 0.00MB + Index: 0.00MB + Engine MyISAM
_DYD_wc_tax_rate_classes: Data: 0.00MB + Index: 0.01MB + Engine MyISAM
_DYD_wc_webhooks: Data: 0.00MB + Index: 0.00MB + Engine MyISAM

### Post Type Counts ###

attachment: 41
custom_css: 1
customize_changeset: 3
nav_menu_item: 20
page: 18
post: 3
product: 33
product_variation: 5
revision: 54
shop_coupon: 9
shop_order: 17
shop_order_refund: 7

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (10) ###

Query Monitor: by John Blackbourn – 3.6.0
Facebook for WooCommerce: by Facebook – 1.11.4 – Installed version not tested with active version of WooCommerce 4.3.0
Jetpack by WordPress.com: by Automattic – 8.6.1
Google Ads for WooCommerce: by Kliken – 1.0.4 – Installed version not tested with active version of WooCommerce 4.3.0
Mailchimp for WooCommerce: by Mailchimp – 2.4.0 – Installed version not tested with active version of WooCommerce 4.3.0
Password Protected: by Ben Huson – 2.3
WooCommerce Beta Tester: by WooCommerce – 2.0.1 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Payments: by Automattic – 1.1.0 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Services: by Automattic – 1.23.2 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce: by Automattic – 4.3.0-beta.1

### Inactive Plugins (18) ###

Akismet Anti-Spam: by Automattic – 4.1.6
Code Snippets: by Code Snippets Pro – 2.14.0
Debug Bar - Enable WP_DEBUG from admin dashboard: by Puvox.software – 1.82
Email Log: by Sudar – 2.3.2
Enable WooCommerce Onboarding Split Test: by Automattic – 1.0.0 – Installed version not tested with active version of WooCommerce 4.3.0
Fusion Builder: by ThemeFusion – 2.1.2
Fusion Core: by ThemeFusion – 4.1.2
Google Analytics for WordPress by MonsterInsights: by MonsterInsights – 7.10.4
Hello Dolly: by Matt Mullenweg – 1.7.2
MOJO Marketplace: by Mike Hansen – 1.4.6
OptinMonster API: by OptinMonster Team – 1.9.8 – Installed version not tested with active version of WooCommerce 4.3.0
VaultPress: by Automattic – 2.1.1
WooCommerce Admin: by WooCommerce – 1.1.3 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Blocks: by Automattic – 2.7.0 – Installed version not tested with active version of WooCommerce 4.3.0
WooCommerce Stripe Gateway: by WooCommerce – 4.4.0 – Installed version not tested with active version of WooCommerce 4.3.0
WordPress Beta Tester: by Peter Westwood
Andy Fragen – 2.2.10

WPForms Lite: by WPForms – 1.6.0.2
WP phpMyAdmin: by Puvox.software – 3.02

### Dropin Plugins (3) ###

db-error.php: db-error.php
db.php: Query Monitor Database Class
maintenance.php: maintenance.php

### Must Use Plugins (3) ###

Endurance Browser Cache: by Mike Hansen – 0.4
Endurance Page Cache: by Mike Hansen – 2.0
SSO: by Garth Mortensen
Mike Hansen – 0.3


### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
variable (variable)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –

### WC Pages ###

Shop base: woocommerce/woocommerce#21 - /
Cart: woocommerce/woocommerce#22 - /cart/
Checkout: woocommerce/woocommerce#23 - /checkout/
My account: woocommerce/woocommerce#24 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Twenty Twenty
Version: 1.4
Author URL: https://wordpress.org/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Action Scheduler ###

Complete: 673
Oldest: 2020-05-18 19:13:10 +0000
Newest: 2020-06-18 18:17:05 +0000

Failed: 1
Oldest: 2020-02-17 19:40:02 +0000
Newest: 2020-02-17 19:40:02 +0000

Pending: 2
Oldest: 2020-06-18 19:17:05 +0000
Newest: 2020-06-18 23:15:39 +0000

`

```
</details>

**Comment by nerrad** [comment]
Thanks for the report. While valid, it looks like this has been around for a while so I edited the title to make it clear this isn’t something introduced by 4.3 (or in this case the version of blocks included with WC 4.3).

**Comment by nerrad** [comment]
This was addressed in #2784
